src/testUtils.tsx

```diff
@@ -5,6 +5,7 @@ import {ApolloProvider, ApolloClient, InMemoryCache, HttpLink} from '@apollo/cli
 import fetch from 'cross-fetch';
 
 import ThemeProvider from 'context/ThemeContext';
+import {create, TestRendererOptions} from 'react-test-renderer';
```

Comment:
There should not be a case where we would need to use this api, please revert the changes in this file